drivers/sensor/nxp/nxp_tmpsns/nxp_tmpsns.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2025 Polytech A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_tmpsns
+
+#include <fsl_tempsensor.h>
+#include <zephyr/drivers/sensor.h>
+
+struct nxp_tmpsns_data {
+	float die_temp;
```

Comment:
If you're going to use a `float` instead of the `q31` format you should depend on `FPU`

Reply:
Will add dependency on FPU